internal/source/code.go

```diff
@@ -111,7 +120,46 @@ func StripComments(sql string) (string, []string, error) {
 			comments = append(comments, t)
 			continue
 		}
+		if strings.HasPrefix(t, "#") {
+			comments = append(comments, strings.TrimPrefix(t, "#"))
+			continue
+		}
 		lines = append(lines, t)
 	}
 	return strings.Join(lines, "\n"), comments, s.Err()
 }
+
+func CleanedComments(rawSQL string, cs CommentSyntax) ([]string, error) {
```

Comment:
We've now implemented comment parsing/cleaning in three places. I was hoping to consolidate but walked it back in favor of a separate code path.

It's some unfortunate technical debt though, since at some point someone is going to complain that we don't support multi-line slash-star comment syntax and we'll have to refactor all of this...

Reply:
So a comment like

```
/*
name: Foo :one
@param bar? text
*/
```

won't work? If that's the case, we'll want to have a fast follow to enable that.

Reply:
Looking over the implementation of CleanedComments, I don't think it's be too hard to add support for multiline comments. But let's address that in a future PR.

Reply:
Yeah the multi-line comment syntax has never worked but there wasn't really a reason for anyone to ask before this change.

Agree we can implement later